packages/wrangler/src/dev/proxy.ts

```diff
@@ -299,13 +308,21 @@ export function usePreviewServer({
 					const usedPort =
 						address && typeof address === "object" ? address.port : port;
 					logger.log(`⬣ Listening at ${localProtocol}://${ip}:${usedPort}`);
-					const accessibleHosts =
-						ip !== "0.0.0.0" ? [ip] : getAccessibleHosts();
+					const accessibleHosts = [];
+					if (ip === "::" || ip === "*" || ip === "0.0.0.0") {
+						accessibleHosts.push(...getAccessibleHosts(true));
+
+						if (ip !== "0.0.0.0") {
+							accessibleHosts.push("localhost");
+							accessibleHosts.push("[::]");
+						}
+					}
 					for (const accessibleHost of accessibleHosts) {
 						logger.log(`- ${localProtocol}://${accessibleHost}:${usedPort}`);
 					}
 				});
-				proxy.server.listen(port, ip);
+
+				proxy.server.listen(port, ip === "*" ? "::" : ip);
```

Comment:
Does `::` always work? Can a user configure their computer to prevent IPV6?

Reply:
I'm not 100% sure, looks like you can disable ipv6 on a computer so I'll test manually and see what happens

Reply:
After testing with
```sh
sysctl -w net.ipv6.conf.all.disable_ipv6=1
sysctl -w net.ipv6.conf.default.disable_ipv6=1
```
manually it seems to still be able to listen on ipv4 addresses